timetomodel/speccing.py

```diff
@@ -132,7 +131,22 @@ def load_series(
         and keyword params which are to be passed into `pandas.Series.resample`.
         For example:
 
-        `resampling_config=dict(closed="left", downsampling_method="sum", upsampling_method="reverse_sum")`
+        ```
+        resampling_config=dict(
+            event_resolution=timedelta(hours=1),
+            closed="left",
+            downsampling_method="sum",
+            upsampling_method="reverse_sum",
+        )
+        ```
+
+        Here, the event resolution describes that each data point denotes a value over a 1-hour period.
```

Comment:
This should be more clear:

`Here, the event resolution describes that, before resampling, each data point denotes ...`

It's not  *after* is it?

I don't understand why this needs to be passed in the resampling config. It's a bit confusing why. Maybe if it s documented better ...

`event_resolution` is also not mentioned in the docstring.

Reply:
I have elaborated the docstring accordingly.